pkg/config/process.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"os"
+	"time"
+)
+
+const (
+	// DefaultGRPCConnectionTimeoutSecs sets the default value for timeout when connecting to the agent
+	DefaultGRPCConnectionTimeoutSecs = 60
+)
+
+func setupProcesses(config Config) {
+	procBindEnvAndSetDefault := func(key string, val interface{}, env string) {
```

Comment:
If you wanted to increase consistency even further you could derive `env` from the `key` argument.

Reply:
Oh good point!